src/Plugin/AiProvider/LlmdAiProvider.php

```diff
@@ -408,14 +428,15 @@ public function maxEmbeddingsInput(string $model_id = ''): int {
   public function embeddingsVectorSize(string $model_id): int {
     // Default vector size mappings for common embedding models.
     // This should be updated based on your LLM-d model configuration.
-    return match($model_id) {
+    return match ($model_id) {
```

Comment:
The orchestrator has changes, so this really shouldn't be a hardcoded list. I think leave it for now, but it should be revisited.